src/layout/questionnaire-list/components/questionnaire.jsx

```diff
@@ -37,28 +37,30 @@ function QuestionnaireListItem({
       <div className="question-list-item-name">
         <span className="glyphicon glyphicon-chevron-right" />
         <Link to={`/questionnaire/${id}`}>{label}</Link>
-        {fusion ? (
-          <input
-            type="radio"
-            name="questionId"
-            onChange={event => handelCheck(event.target.value)}
-            value={id}
-            style={{ height: '20px', width: '20px' }}
-          />
-        ) : (
-          false
-        )}
-        <button
-          type="button"
-          style={{ float: 'right' }}
-          className={BUTTON_DUPLICATE_CLASS}
-          onClick={event => {
-            event.preventDefault();
-            handleOpenPopup(id, label);
-          }}
-        >
-          {Dictionary.duplicate}
-        </button>
+        <div className="check-button">
+          {fusion ? (
+            <input
```

Comment:
You should add an aria-label to this input in order to define a textual alternative.